synced-files/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,84 @@
+# Code of Conduct
+
+_Exercism's most up-to-date Code of Conduct currently resides on [our website](https://exercism.io/code-of-conduct). This version is correct as of 17th Feb 2021._
+
```

Comment:
We can scrap this now

```suggestion
```

Reply:
I updated to the https://github.com/exercism/website-copy/blob/main/pages/code_of_conduct.md version.